include/six.h

```diff
@@ -26,13 +26,19 @@ class Six {
     virtual bool addPythonPath(const char *path) = 0;
     virtual six_gilstate_t GILEnsure() = 0;
     virtual void GILRelease(six_gilstate_t) = 0;
-    virtual bool getCheck(const char *name, const char *init_config, const char *instances, SixPyObject *&check,
-                          char *&version)
+
+    virtual bool getClass(const char *module, SixPyObject *&pyModule, SixPyObject *&pyClass) = 0;
+    virtual bool getClassVersion(SixPyObject *py_class, char *&version) = 0;
```

Comment:
instead of having `getVersion` and `getFile` here, Six could just expose `getAttrString` and we would pass `__version__` and `__file__` from `get_version` and `get_file` in `api.c`